core_lib/helpers/shell_utils.py

```diff
@@ -0,0 +1,78 @@
+from core_lib.helpers.validation import is_int

```

Comment:
this pull request should be into the other branch and not massive=update
that way will not review the same code again 
we can always change the pr target later